packages/palette/src/elements/BarChart/BarChart.tsx

```diff
@@ -3,7 +3,7 @@ import styled from "styled-components"
 import { color, space } from "../../helpers"
 import { ChartTooltipProps, coerceTooltip } from "../DataVis/ChartTooltip"
 import { ProvideMousePosition } from "../DataVis/MousePositionContext"
-import { maxInArray } from "../DataVis/utils/maxInArray"
+import maxBy from "lodash.maxBy"
```

Comment:
@sepans - in Reaction and Force we use `babel-plugin-lodash` to automatically cherry-pick our functions; so rather than add a new dependency we just do 
```js
import { maxBy } from 'lodash'
```
If you can also add https://github.com/artsy/force/blob/master/.babelrc#L26 to palette that would be nice (and would recommend adding it to volt as well.)